### ISSUE TITLE
operator-no-unspaced detects comment as code

### ISSUE THREAD

**Issue by alexander-schranz** [kristerkari/stylelint-scss#310]
The following code:

```scss
// ...
// ...
$breakpoints: (
    laptop: 1200px,     /* 992px - 1200px */
    tablet: 991px,      /* 768px -  991px */
    mobile: 767px,      /*   0px -  767px */
);
```

fails with on current version `3.5.3`:

```
 4:25  ✖  Expected single space before "/"   scss/operator-no-unspaced
 4:25  ✖  Expected single space after "/"    scss/operator-no-unspaced
 4:26  ✖  Expected single space before "*"   scss/operator-no-unspaced
 4:43  ✖  Expected single space after "*"    scss/operator-no-unspaced
 4:44  ✖  Unexpected newline after "/"       scss/operator-no-newline-after
 4:44  ✖  Expected single space before "/"   scss/operator-no-unspaced
 5:25  ✖  Expected single space before "/"   scss/operator-no-unspaced
 5:25  ✖  Expected single space after "/"    scss/operator-no-unspaced
 5:26  ✖  Expected single space before "*"   scss/operator-no-unspaced
 5:34  ✖  Expected single space after "-"    scss/operator-no-unspaced
 5:43  ✖  Expected single space after "*"    scss/operator-no-unspaced
 5:44  ✖  Unexpected newline after "/"       scss/operator-no-newline-after
 5:44  ✖  Expected single space before "/"   scss/operator-no-unspaced
 6:25  ✖  Expected single space before "/"   scss/operator-no-unspaced
 6:25  ✖  Expected single space after "/"    scss/operator-no-unspaced
 6:26  ✖  Expected single space before "*"   scss/operator-no-unspaced
 6:26  ✖  Expected single space after "*"    scss/operator-no-unspaced
 6:34  ✖  Expected single space after "-"    scss/operator-no-unspaced
 6:43  ✖  Expected single space after "*"    scss/operator-no-unspaced
 6:44  ✖  Unexpected newline after "/"       scss/operator-no-newline-after
 6:44  ✖  Expected single space before "/"   scss/operator-no-unspaced
```

This comments should not be detected as operator.

**Comment by kristerkari** [comment]
Thanks! I'll have a look at it.

**Comment by kristerkari** [comment]
Hmm.. I can't get your example to fail in a unit test, and it also does not fail in my test project that I use for debugging stylelint issues.

Is there some other code that you are using addition to your example that could cause it to fail?

**Comment by kristerkari** [comment]
Closing this as inactive. Let's reopen if something new comes up.

**Comment by alexander-schranz** [comment]
@kristerkari  Sorry for the late reply it seems to be an issue with the value-keyword-case:

```
{
  "plugins": [
    "stylelint-scss"
  ],
  "rules": {
      "value-keyword-case": "lower",
      "scss/operator-no-unspaced": true,
      "scss/operator-no-newline-after": true
  }
}
```

```
{
  "name": "test-for-stylelint",
  "version": "1.0.0",
  "scripts": {
    "lint": "stylelint css/*.scss"
  },
  "license": "ISC",
  "devDependencies": {
    "stylelint": "^9.10.1",
    "stylelint-scss": "^3.5.4"
  }
}
```

```
$breakpoints: (
    laptop: 1200px,     /* 992px - 1200px*/
    tablet: 991px,      /* 768px -  991px*/
    mobile: 767px,      /*   0px -  767px*/
);
```

So I think its not really related to stylelint-scss more to stylelint itself as the value-keyword-case does change something which should not be changed:

Related to https://github.com/stylelint/stylelint/issues/3847
